ls1intum/ArTEMiS
# Tests/Update readme with contribution guide + add test stages to travis pipeline

### Checklist

### Motivation and Context



We should integrate our automated tests into the Github documentation and our Travis/Bamboo pipelines.

### Description

1. I restructured the readme file:
- Moved exercise type specific information into a separate folder
- Created contribution area and linked contribution guide to it
2. I added the automated tests to Travis
- For the server tests I had to set a lot of tests to ignore as they were defective, this should be resolved in a consequtive PR

**TODO:**
The following commands need to be added to the Bamboo build plans by a Bamboo admin:
- ./gradlew executeTests
- yarn test

### Steps for Testing


1. Start a Bamboo / Travis build (in this PR)
2. Build should run tests and be successful
